### ISSUE TITLE
Error when pushing a tagged release deployed on Travis

### ISSUE THREAD

**Issue by moustachio-belvedere** [JuliaDocs/Documenter.jl#1330]
Hello,

Our stable documentation was not updating (related to https://github.com/JuliaDocs/Documenter.jl/issues/1257) so I tried to follow the advice given in:

https://discourse.julialang.org/t/how-to-get-documenter-to-recognize-new-release/37111

First by moving our documentation build to a github action, unfortunately our build process is a bit fiddly as we translate some jupyter notebooks to markdown files and, as I am not very knowledgable, I was unable to get things working as a github action. I moved back to the Travis system and using the regex tag trigger mentioned in the above discourse post, Travis now triggers the documentation build job when pushing a tag. However, it yields the following error:

```
┌ Error: Failed to push:
│   exception =
│    ArgumentError: link `"v0.9" => "v0.9.3"` incompatible with link `"v0.9.0" => "v0.9"`.
```

which I haven't been able to diagnose the cause of, and I can't find examples of others running into the same thing. The build on master seemed to be fine. (Note master build can be seen at https://travis-ci.org/github/JuliaRheology/RHEOS.jl/jobs/692344663 and tag push build that failed can be seen at https://travis-ci.org/github/JuliaRheology/RHEOS.jl/jobs/692352295.)

Apologies, this is most likely a problem caused by my mismanagement of docs deployment, but I thought it would probably be appropriate to post here as an issue.

edit: just to clarify also, I deleted the 0.9.3 tag after the build failed as it's only purpose was a 'test' tag for the documentation, it was deleted both locally and remotely.

**Comment by fredrikekre** [comment]
This is because you have a folder https://github.com/JuliaRheology/RHEOS.jl/tree/gh-pages/v0.9 which confuses Documenter.

**Comment by fredrikekre** [comment]
Deleting that and retriggering the CI on the tag build should fix it.

**Comment by moustachio-belvedere** [comment]
Many thanks for getting back to me so quickly. I followed the instruction and deleted that manually. Unfortunately I then got the following error https://travis-ci.org/github/JuliaRheology/RHEOS.jl/jobs/692481138:
```
┌ Error: Failed to push:
│   exception =
│    IOError: symlink: file already exists (EEXIST)
```
I thought it was due to the symlink v0.9.0 so I deleted that in commit https://github.com/JuliaRheology/RHEOS.jl/commit/5962d697ee858a3f79b1ef1aecd010e4ec641cb5 but the error persists.
